fw/t/unit/test_http_parser.c

```diff
@@ -330,75 +326,73 @@ validate_data_fully_parsed(int type)
 	return hm->msg.len == hm_exp_len;
 }
 
-#define TRY_PARSE_EXPECT_PASS(str, type)			\
-({ 								\
-	int _err = type == FUZZ_REQ_H2 ?			\
-		do_h2_parse(h2_buf + H2_HDR_HDR_SZ,		\
-			    h2_len, type) :			\
-		do_split_and_parse(str, type);			\
-	if (_err == TFW_BLOCK || _err == TFW_POSTPONE		\
-	    || !validate_data_fully_parsed(type))		\
-		TEST_FAIL("can't parse %s (code=%d):\n%s",	\
-			  (type == FUZZ_REQ			\
-			   || type == FUZZ_REQ_H2		\
-			   ? "request" : "response"),		\
-			  _err, (str)); 			\
-	__fpu_schedule();					\
-	type == FUZZ_REQ_H2 ? false:				\
-		_err == TFW_PASS;				\
+#define TRY_PARSE_EXPECT_PASS(str, type, chunk_mode)					\
```

Comment:
Please, make sure that on the line and everywhere in PR your line width less than or equals to 81.

Reply:
Ok.